README.md

```diff
@@ -0,0 +1,96 @@
+* **Responsive** - WPPA si adatta a disposiviti sia desktop che mobile, senza perdere alcuna funzionalità.
+* **4 posizioni del menu** - WPPA mette a disposizione 4 posizioni di menu, nelle quali è possibile gestire la selezione della lingua e i canali social.
+* **3 aree widget** - WPPA offre 3 aree dove è possibile inserire widget: una presente in home-page, un'altra nel footer e infine nella sidebar.
+* **Logo e titolo** - WPPA permette la personalizzazione del logo, del titolo e del motto del sito direttamente dalla personalizzazione del tema.
+* **Colori e link** - WPPA consente di selezionare e personalizzare i colori per la testata, il footer e i suoi link.
+* **Home-page** - WPPA ha un template differente per la home-page che ti permette di comunicare al meglio al tuo pubblico.
+* **Pochi file** - WPPA è sviluppato cercando di utilizzare pochi file php nel tema, in modo che possa esser la base per sviluppi propri.
+
+---
+
+## Plugins suggeriti
+
+Per una migliore gestione dei siti su WordPress, consigliamo di installare i seguenti plugins.
+
+#### Integrazione Pubblica Amministrazione
+
++ [Amministrazione Trasparente](https://it.wordpress.org/plugins/amministrazione-trasparente/) - Diffusione di informazioni
++ [Amministrazione Aperta](https://wordpress.org/plugins/amministrazione-aperta/) - Sovvenzioni, contributi e incarichi
++ [Albo Pretorio on-line](https://wordpress.org/plugins/albo-pretorio-on-line/) - Atti in adempimento
++ [WP Spid Italia](https://wordpress.org/plugins/wp-spid-italia/) - Sistema Pubblico di Identità Digitale
+
+#### Funzionalità di base
+
++ [Yoast SEO](https://wordpress.org/plugins/wordpress-seo/) - Gestione SEO avanzata
++ [Contact Form 7](https://wordpress.org/plugins/contact-form-7/) - Creazione form di contatto
++ [dFactory Cookie Notice](https://wordpress.org/plugins/cookie-notice/) - Notifica utilizzo cookies
++ [Akismet](https://wordpress.org/plugins/akismet/) - Anti spam
+
+#### Funzionalità avanzate
+
++ [Download Manager](https://wordpress.org/plugins/download-manager/) - Gestione allegati e documenti
++ [Google Analytics Dashboard](https://wordpress.org/plugins/google-analytics-dashboard-for-wp/) - Integrzione Google Analytics
++ [Page Builder by SiteOrigin](https://wordpress.org/plugins/siteorigin-panels/) - Creazione struttura pagine
++ [Custom Field Suite](https://wordpress.org/plugins/custom-field-suite/) - Creazione di nuovi campi
+
+---
+
+## Segnalazione bug e richieste di aiuto
+Vuoi segnalare un bug o fare una richiesta?
+
+Prima di tutto assicurati che sia un problema relativo al tema Bootstrap Italia e non alla libreria Bootstrap da cui deriva (in tal caso puoi fare riferimento al [repository di Bootstrap](https://github.com/twbs/bootstrap), poi dai un'occhiata a come creare una [issue](https://github.com/italia/bootstrap-italia/blob/master/CONTRIBUTING.md#creare-una-issue).
+
+## Come contribuire
```

Comment:
Toglierei questo capitolo che fa riferimento a Bootstrap Italia, e creerei un CONTRIBUTING.md separato per WPPA.

Reply:
@francescozaia 
Ho riformulato la frase. Comunicare che si può aprire una iusse mi sembra importante, soprattutto in previsione di bug fixing 😅

Reply:
Molto meglio, ma credo che @francescozaia suggerisse anche di non fare un link diretto a boostrap-italia quanto copiare in questo progetto il file contributing :)
(il link è a https://github.com/italia/bootstrap-italia/blob/master/CONTRIBUTING.md#creare-una-issue )